buildSrc/build.gradle

```diff
@@ -16,6 +16,7 @@ repositories {
 }
 
 apply plugin: 'java'
+apply plugin: 'java-gradle-plugin'
```

Comment:
why is this needed?

Reply:
IIRC I had a problem when importing the project with buildship. Tried again and have no problem anymore without that. So seems obsolete and will be removed.

Reply:
still there

Reply:
removed